guacamole/src/main/java/org/apache/guacamole/rest/RESTServiceModule.java

```diff
@@ -83,11 +83,9 @@ protected void configureServlets() {
         bind(AuthenticationService.class);
         bind(AuthTokenGenerator.class).to(SecureRandomAuthTokenGenerator.class);
         bind(DecorationService.class);
-
-        // Automatically translate GuacamoleExceptions for REST methods
-        MethodInterceptor interceptor = new RESTExceptionWrapper();
-        requestInjection(interceptor);
-        bindInterceptor(Matchers.any(), new RESTMethodMatcher(), interceptor);
+        
+        // Get the ExceptionMapper that will rewrite exceptions into JSON.
```

Comment:
`bind()` is not "getting" the `ExceptionMapper`. In this case, the original comment is probably still more appropriate.

Reply:
Copied/pasted the original comment.